website/src/components/Install/index.tsx

```diff
@@ -47,7 +48,9 @@ export const Install: React.FunctionComponent = () => {
             </h4>
 
             <code>
-                <small className={copied ? styles.flashBackground : ''}>{installText}</small>
+                <small className={copied ? classNames(styles.flashBackground, 'text-break') : 'text-break'}>
+                    {installText}
+                </small>
```

Comment:
Just wanted to note that this component has changed a bit in my homepage PR so we'll just have to watch out for merge conflicts.

Reply:
Thanks for the heads up, @bretthayes. This isn't due until April 27, so we have time to incorporate those changes.

Reply:
No worries @zlonko! Yes, already aware of the due date. Just wanted to provide some context here in case one is merged before the other since both tickets are due on the same day.